app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -0,0 +1,32 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class ResponseRedacter
+        ALLOWED_RESPONSE_FIELDS = %w[
+          fraudpoint.score
+          request_id
+          request_result
+          review_status
+          risk_rating
+          summary_risk_score
+          tmx_risk_rating
+        ]
+
+        # @param [String] body
+        def self.redact(response_json)
```

Comment:
Is `@param [String] body` supposed to refer to identify to parameter passed to this function? The names `body` and `response_json` don't match. Also, are we sure it's really a JSON string?

Reply:
switched it to hash.... thanks!